### ISSUE TITLE
add error message on save in case of error

### ISSUE THREAD

**Issue by Mistress-Anna** [spiral-project/formbuilder#39]
I'm using the demo at https://spiral-project.github.io/formbuilder/. When I click on "save form", the browser tries to contact daybed.io which seems blocked due to a CORS issue (from what I see from the js console). However, the site displays no error. The button displays a spinning icon, leading the user to think the saving is in progress, while nothing is happening in reality.

**Comment by Natim** [comment]
Yes the daybed.io demo server has been shut down yesterday. We plan to use a Kinto server instead.

**Comment by almet** [comment]
We should update the code to use [Kinto](https://kinto.readthedocs.org) instead.

Also, having a way to display errors is something we have on the roadmap, but this project hasn't moved since quite some time and needs some love.
